app/services/foreman/renderer/scope/macros/base.rb

```diff
@@ -133,19 +133,22 @@ def save_to_file(filename, content, verbatim: false)
             desc "This is useful when rendering output is a whitespace sensitive format, such as YAML."
             required :count, Integer, desc: 'The number of spaces'
             keyword :skip1, [true, false], desc: 'Skips the first line prefixing, defaults to false', default: false
+            keyword :skip_content, String, desc: 'Skips indentation, if the line equals the given "content", defaults to nil', default: nil
```

Comment:
```suggestion
            keyword :skip_content, String, desc: 'Skips indentation, if the line equals the given content', default: nil
```
Otherwise there is unnecessary duplication:
![ScreenShot-1718028445099](https://github.com/theforeman/foreman/assets/32508194/b6b627cc-8c2e-4225-80f3-5262c1264041)

Reply:
Thanks.